functions/public/scripts/profile.js

```diff
@@ -51,13 +51,49 @@ Profile.prototype.init = function() {
 
 Profile.prototype.authStateObserver = async function (facebookUser) {
   if (facebookUser) {
-    await setUpheaderAndUserData(facebookUser)
+    await setUpheaderAndUserData(facebookUser);
+    await checkProfileSet(facebookUser);
     await setUpProfileCard();
   } else {
     launchHomeScreen();
   }
 };
 
+async function checkProfileSet(facebookUser) {
+    if (user.profileSet) {
+        var dialog = showWarning();
+        if (dialog) {
+            updateUserProfile();
+        } else {
+            showHomeScreen();
+        }
+    }
+}
+
+async function updateUserProfile() {
+    var jsonObject = {};
+    jsonObject["profile_set"] = false;
+    jsonObject["country"] = "";
+    jsonObject["course"] = "";
+    jsonObject["university"] = "";
+    jsonObject["year"] = "";
+
+    const json = JSON.stringify(jsonObject);
+    await httpPatch(`/api/v1/user/` + user.id, json)
+        .then(res => {
+            return JSON.stringify(res);
+        })
+        .catch(error => console.error(error));
+}
+
+function showHomeScreen() {
+    window.location.href = "/";
+}
+
+function showWarning() {
+    return window.confirm("This will reset your profile.");
```

Comment:
Please change `This will reset your profile.` to `This will reset your Profile.`

Reply:
DONE